tests/modules/nf-core/fastqc/main.nf

```diff
@@ -8,8 +8,11 @@ include { FASTQC } from '../../../../modules/nf-core/fastqc/main.nf'
 // Test with single-end data
 //
 workflow test_fastqc_single_end {
-    input = [ [ id:'test', single_end:true ], // meta map
-              [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true) ]
+    input = [
+                [ id:'test', single_end:true ], // meta map
```

Comment:
can prob drop this from the test then, no?

Reply:
Probably, since it amounts to the same as the interleaved test, but may be nice to also have a test for this specific type of data